src/pages/projectSettings/tabs/ContainersTab/getFormSchema.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  CardFieldTemplate,
+  FieldRow,
+} from "components/SpruceForm/FieldTemplates";
+import { SpruceConfig } from "gql/generated/types";
+import { GetFormSchema } from "../types";
+
+export const getFormSchema = (
+  ecsConfig: SpruceConfig["providers"]["aws"]["pod"]["ecs"]
+): ReturnType<GetFormSchema> => ({
+  fields: {},
+  schema: {
+    type: "object" as "object",
+    properties: {
+      containerSizeDefinitions: {
+        title: "Container Configurations",
+        type: "object" as "object",
+        properties: {
+          variables: {
+            type: "array" as "array",
+            title: "",
+            items: {
+              type: "object" as "object",
+              properties: {
+                name: {
+                  type: "string" as "string",
+                  title: "Name",
+                  default: "",
+                  minLength: 1,
+                },
+                memoryMb: {
+                  type: "number" as "number",
+                  title: "Memory (MB)",
+                  minLength: 1,
```

Comment:
Omitting these fields allows me to save and returns a 422 (presumably because they're empty). I think the validation could be improved

<img width="1348" alt="image" src="https://user-images.githubusercontent.com/9298431/234957654-09200fff-06be-414b-ade1-71f3b8bd1446.png">

Reply:
Great catch! 
<img width="802" alt="image" src="https://user-images.githubusercontent.com/4605522/234965147-a5f1e856-6366-4f14-8ccf-270bef6106cc.png">

Reply:
Clearing the default values still allows me to submit and errors (unless there's something else you forgot to push—I never saw those "Value is required" errors)
<img width="1348" alt="image" src="https://user-images.githubusercontent.com/9298431/234976012-b72a6c9b-3e1c-41c2-85aa-3b41cc0c0cee.png">